src/app/node-data/style.scss

```diff
@@ -80,3 +80,7 @@ mat-button-toggle-group[group='operatingSystemGroup'] {
 .additional-margin {
   margin-top: 16px;
 }
+
+.quota-widget {
+  margin-bottom: 22px !important;
```

Comment:
Can you please share the reason for using `!important`?

Reply:
The margin here was being overridden by the `fxLayoutGap`, so I had to use `!important` here
<img width="538" alt="image" src="https://user-images.githubusercontent.com/56511988/194860632-88aa5f20-ea55-484a-b906-b524813197ed.png">